bat/tests/01-bundle-commands/run.bats

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env bats
+
+# shared test functions
+load ../../lib/mixerlib
+
+@test "Initialize a mix at version 10" {
+  mixer init --clear-version $CLRVER --mix-version 10
+  [[ -f $BATS_TEST_DIRNAME/builder.conf ]]
+  [[ -f $BATS_TEST_DIRNAME/mixbundles ]]
+  [[ $(wc -l < $BATS_TEST_DIRNAME/mixbundles) == 4 ]]
+  [[ -d $BATS_TEST_DIRNAME/local-bundles ]]
+  [[ -d $BATS_TEST_DIRNAME/upstream-bundles/clr-bundles-$CLRVER/bundles ]]
+}
+
+@test "List the bundles in the mix" {
+  run mixer bundle list
+  [[ ${#lines[@]} == 4 ]]               # Exactly 4 bundles in the mix
+  [[ "$output" =~ os-core[[:space:]] ]] # To avoid just matching os-core-update
+  [[ "$output" =~ os-core-update ]]
+  [[ "$output" =~ kernel-native ]]
+  [[ "$output" =~ bootloader ]]
+}
+
+@test "Add an upstream bundle to the mix" {
+  mixer bundle add editors
+
+  run mixer bundle list
+  [[ ${#lines[@]} > 4 ]]                           # More bundles in list now
+  [[ "$output" =~ editors[[:space:]]+\(upstream ]] # "editors" bundle is from upstream
```

Comment:
Do we really care about the `(upstream)` tag here? My thought is that that could change, for instance we may want to say `[upstream]` in the future or something, so we should only test the functional information (`editors`)

Reply:
I don't care that there are parens around it, but I do care that it reports being from upstream.  I didn't spend a great amount of time on this regex, but I put the paren in (instead of, say, `.*` or something) because I didn't want to match
```
editors
otherbundle   (upstream)
```